python/lsst/validate/drp/validate.py

```diff
@@ -97,7 +97,8 @@ def get_filter_name_from_job(job):
     return filter_name
 
 
-def run(repo_or_json, metrics=None, makePrint=True, makePlot=True,
+def run(repo_or_json, metrics=None,
+        outputPrefix=None, makePrint=True, makePlot=True,
```

Comment:
Defaulting to `None` makes this methods signature different from all the other methods which are now defaulting to `""`.